Paladin.cpp

```diff
@@ -1,7 +1,24 @@
 #include "Paladin.h"
+#include "Utility.h"
 
-// Paladin::Paladin
+Paladin::Paladin(const std::string& name_, int hp_, int armor_) : Character(hp_, armor_, 10), name(name_)
+{
+    helpfulItems = makeHelpfulItems(9);
+    defensiveItems = makeDefensiveItems(13);
+}
 
-// Paladin::getName
+void Paladin::attack(Character& other)
+{
+    std::cout << name << " is attacking " << other.getName() << "!!!" << std::endl;
+    Character::attack(other);
+}
```

Comment:
This function is unneeded.
The base class implementation of `attack` is fine
- [ ] remove this function
- [ ] revise